src/GridFS/ReadableStream.php

```diff
@@ -48,13 +51,13 @@ class ReadableStream
     /** @var integer */
     private $chunkOffset = 0;
 
-    /** @var CursorInterface|null */
```

Comment:
This dates back to https://github.com/mongodb/mongo-php-library/commit/3c6b5f2999574d3e8d479195363a7ee241e1a584#diff-15b1c36af88c266bd8e54d1a2d3b409e6e29076f89c75f4dae0ec1c8538a787eR51 from [PHPLIB-598](https://jira.mongodb.org/browse/PHPLIB-598).

When the original IteratorIterator hint was replaced with CursorInterface, this became inconsistent with how we utilize `$chunksIterator` later in the class. CursorInterface provides a few methods (e.g. `setTypeMap()`) and extends Traversable, which wouldn't have guaranteed the iterator methods we later invoke.

[PHPC-1123](https://jira.mongodb.org/browse/PHPC-1123) added CursorInterface in 1.6.0 but the iterator methods didn't arrive until [PHPC-1691](https://jira.mongodb.org/browse/PHPC-1691) in 1.9.0 (when PHP 8.0 required them). And PHPLIB-598 was downstream work for PHPC-1691.

I suppose the original CursorInterface hint was just a mistake and PHPLIB-598 should have used Cursor. Pretty neat that we're just now catching this thanks to psalm.